application/src/main/java/org/thingsboard/server/service/ws/DefaultWebSocketService.java

```diff
@@ -545,7 +545,7 @@ public void onSuccess(List<AttributeKvEntry> data) {
                 TbAttributeSubscription sub = TbAttributeSubscription.builder()
                         .serviceId(serviceId)
                         .sessionId(sessionId)
-                        .subscriptionId(cmd.getCmdId())
+                        .subscriptionId(sessionRef.getSessionSubIdSeq().incrementAndGet())
```

Comment:
What about notification subscriptions? We are planning on adding notification widget, so I guess we should already do it for notification subs as well.

Reply:
The bug was while adding the TbSubscription to the set in TbEntityLocalSubsInfo
```
private final Set<TbSubscription<?>> subs = ConcurrentHashMap.newKeySet();
```

TbSubscription hashCode/equals uses TbSubscriptionType so we are safe in such cases.